### PR TITLE
AT_02.06_11 | Homepage (Dashboard) > Adding Main panel description

### PR DESCRIPTION
**Pull request by mborovaya**
https://trello.com/c/VoLe8FgR/942-at020611-homepage-dashboard-adding-main-panel-description

### CODE REVIEW

File: cypress/e2e/homepageDescription.cy.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress"/>
+
+describe('<Homepage (Dashboard) > Main panel description',()=>{
+    const addDescription = 'Add description'
+    const messageInDescription = 'This is a description'
+    const editDescription = 'Edit description'
```
Comment (Rom-T) [inline cypress/e2e/homepageDescription.cy.js:6]: Please move data in the fixtures

File: cypress/e2e/homepageDescription.cy.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress"/>
+
+describe('<Homepage (Dashboard) > Main panel description',()=>{
```
Comment (Rom-T) [inline cypress/e2e/homepageDescription.cy.js:3]: Please name the describe as the name of your file.

File: cypress/e2e/homepageDescription.cy.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    const addDescription = 'Add description'
+    const messageInDescription = 'This is a description'
+    const editDescription = 'Edit description'
+
+    it('AT_02.06_11 | Homepage (Dashboard) > Adding Main panel description', () => {
+        cy.get('#description-link').should('contain', addDescription).click();
+        cy.get('#description-link').should('not.exist');
+        cy.get('[name="description"]').type(messageInDescription);
+        cy.get('.jenkins-button[formnovalidate="formNoValidate"]').should('have.text', 'Save').click();
```
Comment (Rom-T) [inline cypress/e2e/homepageDescription.cy.js:12]: Please move `'Save'` in the fixtures

File: cypress/e2e/homepageDescription.cy.js (hunk 1 above)
Comment (mborovaya) [reply cypress/e2e/homepageDescription.cy.js]: Thanks! Done

File: cypress/e2e/homepageDescription.cy.js (hunk 2 above)
Comment (mborovaya) [reply cypress/e2e/homepageDescription.cy.js]: Thanks! Done

File: cypress/e2e/homepageDescription.cy.js (hunk 3 above)
Comment (mborovaya) [reply cypress/e2e/homepageDescription.cy.js]: Thanks! Done

File: cypress/e2e/homepageMainPanelDescription.cy.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress"/>
+import * as descriptions from '../fixtures/descriptionsProject.json';
```
Comment (Rom-T) [inline cypress/e2e/homepageMainPanelDescription.cy.js:2]: Can be simply done
`import descriptions from '../fixtures/descriptionsProject.json';`

Comment (mborovaya) [reply cypress/e2e/homepageMainPanelDescription.cy.js]: Thank you! Changed and it works

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by IrynaKolyada — APPROVED** [review]

**Pull request merged**